src/components/Modal/Modal.tsx

```diff
@@ -88,11 +115,11 @@ class Modal extends PureComponent {
     return createPortal(
       <div
         className={cn("modal-overlay fixed inset-0 bg-black-o-75 z-50", {
-          "opacity-0": this.isAboutToClose
+          "opacity-0": isAboutToClose
         })}
-        onMouseUp={this.gracefulClose}
+        onMouseUp={(e) => this.gracefulClose(e)}
```

Comment:
**question** Any reason why we're not using `onMouseUp={this.gracefulClose}`?

Reply:
well I did a mistake, that "e" parameter should be optional because is not used in "onMouseUp" but used into "onEscape" method.